src/sentry/incidents/subscription_processor.py

```diff
@@ -212,11 +212,20 @@ def get_comparison_aggregation_value(
                     "end": end,
                 },
             )
-            time_col = ENTITY_TIME_COLUMNS[get_entity_key_from_query_builder(query_builder)]
+            entity_key = get_entity_key_from_query_builder(query_builder)
```

Comment:
I think you want to have an [Entity](https://github.com/getsentry/snuba-sdk/blob/65e490a481af00c81693dc590632bd0d9db4697f/snuba_sdk/entity.py#L16), not an [EntityKey](https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/dataset.py#L63).

Reply:
Yea - the `EntityKey.value` gets translated into the `Entity` below